packages/core/scripts/local/getMedianHistoricalPrice.js

```diff
@@ -19,7 +19,9 @@ const UMIP_PRECISION = {
   BTCDOM: 2,
   ALTDOM: 2,
   BCHNBTC: 8,
-  "GASETH-TWAP-1Mx1M": 18
+  "GASETH-TWAP-1Mx1M": 18,
```

Comment:
So we need to return these prices up to 18 decimals of precision?

Reply:
Yep, that's what's specified in the UMIP (or at least that's how I understood it)

Reply:
Yeah that is how it is specified, but `GASETH-TWAP-1Mx1M` doesn't use the `getMedianHistoricalPrice` script. No harm in including it though.